rs/sns/governance/src/governance.rs

```diff
@@ -4667,7 +4669,7 @@ impl Governance {
         let (current_version, sns_governance_canister_id) = {
             let now = self.env.now();
             let Some(current_version) = self.proto.deployed_version.clone() else {
-                // TODO: add logs
+                log!(INFO, "Deployed_version not set - should be impossible");
```

Comment:
If this really should be impossible, then let's make this err-level log

Would this be expected to happen in testing?